derp/derp_server.go

```diff
@@ -1187,6 +1187,10 @@ func (s *Server) verifyClient(ctx context.Context, clientKey key.NodePublic, inf
 			return fmt.Errorf("peer %v not authorized (not found in local tailscaled)", clientKey)
 		}
 		if err != nil {
+			if strings.Contains(err.Error(), "invalid 'addr' parameter") {
```

Comment:
Should we have the derper check the local tailscaled version when it starts?

Reply:
I was afraid of that being too fragile. People build these things in all sorts of ways and with e.g. a github tarball download without a git directory, it doesn't even include version info in the resulting binary.

Reply:
But, yes, if we can get it working reliably (or maybe re-use `tailcfg.CurrentCapabilityVersion` and send that in Request+Response HTTP headers and if they ever differ and there's any other error, suggest that the real problem is the skew)